resources/views/categories/index.blade.php

```diff
@@ -0,0 +1,113 @@
+@extends('layouts.app')
+@section('page-title')
+  @lang('categories.index.title')
+@endsection
+
+@section('section-title')
+    @lang('categories.index.section-title')
+@endsection
+
+@section('page-content')
+    <div class="x_content">
+        <table id="list-categories-table" class="table table-striped jambo_table table-bordered">
+          <thead>
+            <tr class="headings">
+              <th class="column-title text-center" style="width: 5%">#</th>
```

Comment:
[rank B] did you read my previous comment? Please researching about the datatable to adjust width size of columns.